about/policy/sgid/index.md

```diff
@@ -10,60 +10,107 @@ categories: []
+- The entity providing the dataset should be the **authoritative** or original source (for example, a county's parcels must come from the County government).
+- The entity providing the dataset does not have the internal **capability** to publicly serve the data (such as through an existing ArcGIS Online organization). An entity's data storage costs or server capacity may not be sufficient justification for adding data to the SGID.
 
-Additional, related info:
+Agencies and organizations can still share datasets that don't meet the guidelines above with our [Open Data site](https://opendata.gis.utah.gov/). This is a great way to share data mainly intended for the general public with a wide audience. Please see our [Open Data sharing guidelines]({%link gis-data-sharing/open-data/index.md %}) for more information.
 
-A corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1) will be added, and will contain at a minimum:
+We will periodically review the SGID for datasets that no longer meet these criteria. Specifically, we may consider removing datasets that see little to no usage for a prolonged period of time.
+
+### Dataset Introduction Process
+{: .text-left }
+
+We will use the [Porter](https://github.com/agrc/porter) project to guide and track the steps needed to add or make breaking changes (remove, schema change, etc.) to products contained in the SGID. A brief overview of these steps is included below.
+
+We will add a corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1). This will contain at a minimum:
 
 - The authoritative access from location
 - The table name
 - A description
 - The data steward
 - The steward contact name and email
-- A link the the data page on the AGRC website
+- A link to the data page on the AGRC website
 - A link to the SGID Open Data page
 
-The ArcGIS Online Item will reside in the corresponding ISO category folder and contain the appropriate tags. If the data is authoritative, it will be marked as `authoritative`. If the data does not receive updates, it will be tagged as `static`. If the data has historical relevance **but is not the most recent data** it will be tagged as `shelved`. Note: the [AGOL Validator](https://github.com/agrc/agol-validator) project will transfer metadata from the Internal SGID database, including summary, description, and use limitations.
+We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding ISO category folder and group and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`.
```

Comment:
remove ISO mention
```suggestion
We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding category folder and group and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`.
```